client/webpack.config.js

```diff
@@ -13,6 +13,6 @@ module.exports = withDefaults({
     },
     output: {
         filename: 'extension.js',
-        path: path.join(__dirname, 'dist'),
+        path: path.join(__dirname, 'out'),
```

Comment:
apparently dist is changed to out. Dist is removed from the ignore files, but 'out' is not added to the ignore files. Is that correct?

Reply:
Yes, it is. The transpiled javascript is output to the `out` folders, instead of `dist`. This means there will not be a `dist` folder anymore, so it was removed. The `out` folder was already in most ignore files. In `.vscodeignore`, the out folders are included in `client` and `server`, but the compiled client and server code is un-ignored by `!client/out/*.js` and `!server/out/*.js`.